### ISSUE TITLE
Expecting the training code

### ISSUE THREAD

**Issue by jackieyung** [yohanshin/WHAM#37]
@dalgu90 @yohanshin @RohaanA Thanks for your amazing method. 

I'm very expecting the training code release. I cannot wait for training the model on my own dataset. So, when could you release the training code? Thanks a lot.

**Comment by yohanshin** [comment]
Hi @jackieyung ,

Thank you for your interest in our work! Yes, I am also greatly looking forward to sharing my intellectual findings with people in the community and will definitely share the training code for reproducibility. Since I am handling some other urgent tasks for now, it is quite difficult to promise a hard deadline for the release, but I am expecting to have it done in the early February. Before then, I am always happy to answer any concerns/questions regarding the training pipeline. Thanks!

**Comment by jackieyung** [comment]
Hi, @yohanshin, how time flies! would you please release your training code?  I'm just really need your training code to train my own model in my own dataset. Thanks a lot.

**Comment by yohanshin** [comment]
Hi @jackieyung 

I sincerely apologize for the delayed release of the training code. I had some unexpected tasks during the last couple of weeks. I will provide the training script as soon as possible.

**Comment by yxyyundanfengqing** [comment]
Thank you very much for your excellent work, I am looking forward to the release of your training code.

**Comment by IsshikiHugh** [comment]
Hi @yohanshin, thanks for you great work!
But may I ask when the training code might be released? Will it be released before the middle of March? I just want to know a general situation so that I can adjust my plans.
Thanks for your reply in advance! Good day to you!

**Comment by jackieyung** [comment]
> Hi @jackieyung
> 
> I sincerely apologize for the delayed release of the training code. I had some unexpected tasks during the last couple of weeks. I will provide the training script as soon as possible.

Hi,  @yohanshin，how time flies!  The another weeks passed by, would you please give us a time plan when to release the training code? Your method is so amazing that we cannot wait to trying it on our own dataset.

We are looking forword to your reply. Thanks a lot.

**Comment by yxyyundanfengqing** [comment]
@yohanshin I greatly appreciate your impressive work !
In the project's README, you mentioned plans to release the training code. If possible, could you provide a more specific timeline for when the training code will be available?
Thank you for your contributions and dedication to this field.

**Comment by yohanshin** [comment]
Hi @jackieyung and @yxyyundanfengqing . I sincerely apologize for the delayed release of the training code. I am putting my effort into the training script release and now `stage1` training is released at this [branch](https://github.com/yohanshin/WHAM/tree/train). Please refer to the training code (mostly borrowed from VIBE and the main contributions are in loss functions and synthetic data generation).

because I haven't run the entire training after migrating the code into the cleaned framework, the code might undergo small changes afterward. The training on the video datasets should be relatively simpler, so I will update it soon (during next week). Once I release all training scripts, I will merge them into the main branch. Please do not hesitate to ask questions through the issue or [email](mailto:soyongs@andrew.cmu.edu) for anything related to the training code.

**Comment by yxyyundanfengqing** [comment]
HI @yohanshin 
Thank you so much for your prompt reply and for kindly sharing the training code!

**Comment by jackieyung** [comment]
> Hi @jackieyung and @yxyyundanfengqing . I sincerely apologize for the delayed release of the training code. I am putting my effort into the training script release and now `stage1` training is released at this [branch](https://github.com/yohanshin/WHAM/tree/train). Please refer to the training code (mostly borrowed from VIBE and the main contributions are in loss functions and synthetic data generation).
> 
> because I haven't run the entire training after migrating the code into the cleaned framework, the code might undergo small changes afterward. The training on the video datasets should be relatively simpler, so I will update it soon (during next week). Once I release all training scripts, I will merge them into the main branch. Please do not hesitate to ask questions through the issue or [email](mailto:soyongs@andrew.cmu.edu) for anything related to the training code.

Thanks a lot for your training code sharing. That's so nice of you!

**Comment by yxyyundanfengqing** [comment]
Hi@yohanshin 
I have a couple of questions regarding the AMASS dataset used for training:Could you please confirm if the AMASS training data is in the SMPL+H format? Additionally, I was wondering which specific subsets of AMASS were used for training? For example, was the ACCAD dataset included?
Thanks a lot ！

**Comment by IsshikiHugh** [comment]
> Hi@yohanshin I have a couple of questions regarding the AMASS dataset used for training:Could you please confirm if the AMASS training data is in the SMPL+H format? Additionally, I was wondering which specific subsets of AMASS were used for training? For example, was the ACCAD dataset included? Thanks a lot ！

I met the same problems and I try to run the smallest subset of AMASS, [SSM](https://amass.is.tue.mpg.de/) with SMPL-X G, and found that the `3dpw_val_vit.pth` is missing. And I just **temporarily** rename the `3dpw_test_vit.pth` to it and successfully run it.

I don't know whether it's right or not but it just works. Maybe these words can temporarily helps you before the author replies.

**Comment by yxyyundanfengqing** [comment]
> > Hi@yohanshin I have a couple of questions regarding the AMASS dataset used for training:Could you please confirm if the AMASS training data is in the SMPL+H format? Additionally, I was wondering which specific subsets of AMASS were used for training? For example, was the ACCAD dataset included? Thanks a lot ！
> 
> I met the same problems and I try to run the smallest subset of AMASS, [SSM](https://amass.is.tue.mpg.de/) with SMPL-X G, and found that the `3dpw_val_vit.pth` is missing. And I just **temporarily** rename the `3dpw_test_vit.pth` to it and successfully run it.
> 
> I don't know whether it's right or not but it just works. Maybe these words can temporarily helps you before the author replies.

@IsshikiHugh  Thank you very much! I will try your solution.
